src/view/util/CaseDataInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+interface Props {
+  lastUpdated?: string;
+}
+
+const CaseDataInfo: React.FC<Props> = props => {
+  if (!props.lastUpdated) {
+    return null;
+  }
+
+  if (Number.isNaN(Date.parse(props.lastUpdated))) {
+    console.error(`Invalid date: ${props.lastUpdated}`);
+    return null;
+  }
+
+  const caseAgeEnd = new Date(props.lastUpdated);
+  const caseAgeStart = new Date(caseAgeEnd);
+  caseAgeStart.setFullYear(caseAgeEnd.getFullYear() - 1);
+
+  return (
+    <React.Fragment>
+      <p>
+        Showing cases from: {caseAgeStart.toLocaleString()} -{" "}
+        {caseAgeEnd.toLocaleString()}
+      </p>
+      <p>Excluding substatus: BCU and FDNS</p>
```

Comment:
I would probably just say "Excluding cases that have been referred to BCU or FDNS" rather than getting into the mechanics. (But we want to get into the mechanics, should we enumerate the specific substatuses? They might not ask, and I don't think it much matters, but we could just tell them anyway.)